plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/model/LayoutOptionUIData.xtend

```diff
@@ -0,0 +1,133 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ * 
+ * Copyright 2019 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ * 
+ * This code is provided under the terms of the Eclipse Public License (EPL).
+ */
+package de.cau.cs.kieler.klighd.lsp.model
+
+import java.util.ArrayList
+import java.util.Collection
+import java.util.List
+import org.eclipse.elk.core.data.LayoutOptionData
+import org.eclipse.elk.core.data.LayoutOptionData.Type
+import org.eclipse.xtend.lib.annotations.Data
+
+/**
+ * Data class holding a representation of all necessary data for some arbitrary UI displaying layout options.
+ * This is similar to the {@link LayoutOptionData}, just in a serializable form for transmission to clients.
+ * 
+ * @author nre
+ */
+@Data
+class LayoutOptionUIData {
+    
+    /** The default value for the lower bound. */
+    static final int DEFAULT_MIN = 0
+
+    /** The default value for the upper bound. */
+    static final int DEFAULT_MAX = 100
+    
```

Comment:
why these as min/max values?

Reply:
They are the same in klighd.ui.internal.options.LayoutOptionControlFactory#DEFAULT_MIN and #DEFAULT_MAX. We simply cannot display +/- infinity on any slider.